benchmarks/cugraph/standalone/cugraph_bulk_sampling.py

```diff
@@ -0,0 +1,617 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
```

Comment:
I think we can probably separate this file and merge it as separate PR ?

Reply:
Yes, I'll take care of that today.